app/models/delivery.rb

```diff
@@ -0,0 +1,17 @@
+# == Schema Information
+#
+# Table name: deliveries
+#
+#  id                    :integer          not null, primary key
+#  arrived               :boolean
+#  description           :string
+#  details               :text
+#  supposed_to_arrive_on :date
+#  created_at            :datetime         not null
+#  updated_at            :datetime         not null
+#  user_id               :integer
+#
+
+class Delivery < ApplicationRecord
+  belongs_to :user, class_name: "User", foreign_key: "user_id"
```

Comment:
If the class name and foreign key are all the same, you can just write "belongs_to :user" and it works the same :)